### PR TITLE
Some utils for doing data manipulation

### PR DESCRIPTION
**Pull request by why-not-try-calmer**

### CODE REVIEW

File: defrag/modules/utils.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+# Defrag - centralized API for the openSUSE Infrastructure
```
Comment (KaratekHD) [inline defrag/modules/utils.py:1]: I think it's a bad idea to have these utils as a module. Could you move them to a `helpers` or `utils` folder similar to #6 ?

**Review by KaratekHD — CHANGES_REQUESTED** [review]

File: defrag/modules/utils.py (hunk 1 above)
Comment (why-not-try-calmer) [reply defrag/modules/utils.py]: did that

Comment (why-not-try-calmer) [reply defrag/modules/utils.py]: It is ok for you now or you want me to change something? Asking because this PR is marked as "Change requested" still.

**Review by KaratekHD — APPROVED** [review]
Looks good

**Comment by why-not-try-calmer** [comment]
Alright, thanks!

**Pull request merged**